ResourceManager/Xliff/TranslationFile.cs

```diff
@@ -48,7 +48,7 @@ public TranslationCategory FindOrAddTranslationCategory(string translationCatego
         public void AddTranslationCategory(TranslationCategory translationCategory)
         {
             if (string.IsNullOrEmpty(translationCategory.Name))
-                new InvalidOperationException("Cannot add translationCategory without name");
+                throw new InvalidOperationException("Cannot add translationCategory without name");
```

Comment:
This break test http://teamcity.codebetter.com/viewLog.html?buildId=217387&buildTypeId=GitExtensions_PullRequests
Also change this:
```
diff --git a/ResourceManager/TranslationUtils.cs b/ResourceManager/TranslationUtils.cs
index 606ad4d..a46cd1f 100644
--- a/ResourceManager/TranslationUtils.cs
+++ b/ResourceManager/TranslationUtils.cs
@@ -17,7 +17,8 @@ public static void AddTranslationItem(string category, object obj, string proper
 
         public static void AddTranslationItemsFromFields(string category, object obj, ITranslation translation)
         {
-            Xliff.TranslationUtl.AddTranslationItemsFromFields(category, obj, translation);
+            if (!String.IsNullOrEmpty(category))
+                Xliff.TranslationUtl.AddTranslationItemsFromFields(category, obj, translation);
         }
 
         public static void AddTranslationItemsFromList(string category, ITranslation translation, IEnumerable<Tuple<string, object>> items)
@@ -37,7 +38,8 @@ public static void TranslateItemsFromList(string category, ITranslation translat
 
         public static void TranslateItemsFromFields(string category, object obj, ITranslation translation)
         {
-            Xliff.TranslationUtl.TranslateItemsFromFields(category, obj, translation);
+            if (!String.IsNullOrEmpty(category))
+                Xliff.TranslationUtl.TranslateItemsFromFields(category, obj, translation);
         }
     }
 }
```